scripts/auto-stop-idle/autostop.py

```diff
@@ -96,21 +96,26 @@ def get_notebook_name():
                 if notebook['kernel']['connections'] == 0:
                     if not is_idle(notebook['kernel']['last_activity']):
                         idle = False
+                        print('Notebook idle state is now: ', idle)
                 else:
                     idle = False
+                    print('Notebook idle state is now: ', idle)
             else:
                 if not is_idle(notebook['kernel']['last_activity']):
                     idle = False
+                    print('Notebook idle state is now: ', idle)
         else:
             print('Notebook is not idle:', notebook['kernel']['execution_state'])
             idle = False
+            print('Notebook idle state is now: ', idle)
```

Comment:
Doesn't seems to add value - there's already a print statement 2 lines above.

Reply:
the goal is to print when the idle state is toggled off. the function prints the current state of the notebook. updating the statements to provide more descriptive value.